### ISSUE TITLE
Using HTML2PDF, text taking extra space

### ISSUE THREAD

**Issue by RoyNidhi** [eKoopmans/html2pdf.js#689]
I am using the HTML2PDF library, and on the click of the button, a pdf shows up to download. In that PDF, I'm getting extra space for some text only.

HTML:-

<p className="flex flex flex-between chip-item">
     <span className="medium-text">Vehicle Required: </span>
     <span>Bike</span>
</p>

CSS:- 
.chip-item{
    padding: 11px;
    font: 400 14px "GeneralSans-Regular", Arial, Sans-Serif;
    color: #23201c;
    background: #f8d834;
    border-radius: 20px;
    min-height: 36px;
    display: flex;
}

![image](https://github.com/eKoopmans/html2pdf.js/assets/106378562/21a9ec3c-e52d-498d-b48a-1a25f5077604)

**Comment by MoussaPro** [comment]
Did u solve this?

**Comment by RoyNidhi** [comment]
Yeah, It's because of the font size. I've changed the font size to 16px and then it worked fine.

**Comment by ibukunAce** [comment]
i am still having this issue
